src/Stratis.Bitcoin.Networks/ProtocolVersion/ProtocolVersion.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Networks
```

Comment:
namespace not corresponding to file location

Reply:
ok